capa/helpers.py

```diff
@@ -35,3 +37,9 @@ def is_runtime_ida():
 
 def assert_never(value: NoReturn) -> NoReturn:
     assert False, f"Unhandled value: {value} ({type(value).__name__})"
+
+
+def is_dotnet_file(pe: PE) -> bool:
```

Comment:
using pefile here, could use dnfile?

Reply:
hmm good question. dnfile invokes pefile so currently we are invoking pefile's parsing twice (when processing .NET PEs). I'm unsure if this is a performance hit great enough to warrant complicating main with more logic. I'd say for now we don't invoke dnfile until we know we are processing a .NET PE. So IMO using pefile here is fine.